docs/contracts/StMatic.md

```diff
@@ -17,222 +17,123 @@ stMATIC will be integrated in variety of DeFi applications across Ethereum and P
 ```
+---
 
-:::note
-Balances are dynamic and equal the `_account`'s share in the amount of the
-total Ether controlled by the protocol. See `sharesOf`.
-:::
-
-### getTotalShares()
-
-Returns the total amount of shares in existence.
-
-```sol
-function getTotalShares() returns (uint256)
-```
-
-### sharesOf()
-
-Returns the amount of shares owned by `_account`
-
-```sol
-function sharesOf(address _account) returns (uint256)
-```
-
-### getSharesByPooledEth()
-
-Returns the amount of shares that corresponds to `_ethAmount` protocol-controlled Ether
-
-```sol
-function getSharesByPooledEth(uint256 _ethAmount) returns (uint256)
-```
-
-### getPooledEthByShares()
-
-Returns the amount of Ether that corresponds to `_sharesAmount` token shares
-
-```sol
-function getPooledEthByShares(uint256 _sharesAmount) returns (uint256)
-```
-
-### getFee()
-
-Returns staking rewards fee rate
-
-```sol
-function getFee() returns (uint16)
-```
-
-#### Returns:
-
-Fee in basis points. 10000 BP corresponding to 100%.
-
-### getFeeDistribution()
-
-Returns fee distribution proportion
-
-```sol
-function getFeeDistribution() returns (
-  uint16 treasuryFeeBasisPoints,
-  uint16 insuranceFeeBasisPoints,
-  uint16 operatorsFeeBasisPoints
-)
-```
-
-#### Returns:
-
-| Name                      | Type     | Description                                                                            |
-| ------------------------- | -------- | -------------------------------------------------------------------------------------- |
-| `treasuryFeeBasisPoints`  | `uint16` | Fee for the treasury. Expressed in basis points, 10000 BP corresponding to 100%.       |
-| `insuranceFeeBasisPoints` | `uint16` | Fee for the insurance fund. Expressed in basis points, 10000 BP corresponding to 100%. |
-| `operatorsFeeBasisPoints` | `uint16` | Fee for the node operators. Expressed in basis points, 10000 BP corresponding to 100%. |
-
-### getWithdrawalCredentials()
-
-Returns current credentials to withdraw ETH on ETH 2.0 side after the phase 2 is launched
-
-```sol
-function getWithdrawalCredentials() returns (bytes32)
+### getTotalStake()
+Returns the total amount of stake the StMatic contract has in a validator
+```solidity
+function getTotalStake(IValidatorShare _validatorShare) returns (uint256, uint256)
 ```
+#### Parameters:
 
-### getBufferedEther()
-
-Get the amount of Ether temporary buffered on this contract balance
-
-:::note
-
-Buffered balance is kept on the contract from the moment the funds are received from user
-until the moment they are actually sent to the official Deposit contract.
-
-:::
+| Name         | Type      | Description                  |
+| ------------ | --------- | ---------------------------- |
+| `_validatorShare` | `ValidatorShare` | An instance of the validator share contract  |
+---
 
-```sol
-function getBufferedEther()  returns (uint256)
+### getLiquidRewards()
+Returns the liquid reward the StMatic contract has in a validator
```

Comment:
Returns the reward accumulated by the StMATIC contract in a specific validator share.